rootfs/scheduler/__init__.py

```diff
@@ -601,11 +601,14 @@ def _create_rc(self, name, image, command, **kwargs):
         num = kwargs.get('num', {})
         imgurl = self.registry + "/" + image
         TEMPLATE = RCD_TEMPLATE
-        shalen = len(image[image.index(":")+5:])
-        git = image[image.index(":")+1:image.index(":")+4]
-        if git == "git" and shalen == 8 and app_type == 'web':
-            imgurl = "http://"+settings.S3EP+"/git/home/"+image+"/push/slug.tgz"
+
+        # Check if it is a slug builder image.
+        # Example format: golden-earrings:git-5450cbcdaaf9afe6fadd219c94ac9c449bd62413s
+        vcs, sha = image[image.index(':')+1:].split('-')
```

Comment:
yeah, we can do it as part of another PR. Create a ticket. That should take care of a few hardcoded points we have in the system as it stands